docs/source/federation/production.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Running Federation in production
+description: Ensuring reliability and robustness
+---
+
+Like any distributed architecture, you should make sure that your federated GraphQL layer has proper observability, monitoring, and automation to ensure reliability and performance of both your gateway and the federated services underneath it. Serving your GraphQL API from a distributed architecture has many benefits, like productivity, isolation, and being able to match the right services with the right runtimes. Operating a distributed system also has more complexity and points of failure than operating a monolith, and with that complexity comes a need to heighten observability into the state of your system and control over its coordination.
```

Comment:
We should change language here to warn against using federation w/ only one team (i.e. federation has benefits for coordinating across multiple teams, and shouldn't be used otherwise)